Samples/FilterOnObservableCondition/MainView.js

```diff
@@ -35,11 +35,11 @@ var items = Observable(
 	new Color('#FFFF00', 'Other 5', COLOR_TYPES.OTHER)
 );
 
-var filteredItems = currentCategory.map(function(category){
```

Comment:
This example is also shown here: https://www.fusetools.com/docs/fusejs/observable#where-condition

Perhaps it's correct _not_ to use `flatMap` here, or? I'm actually not sure